crates/sargon/src/system/sargon_os/transactions/support/signable_manifest_summary.rs

```diff
@@ -0,0 +1,151 @@
+use crate::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct SignableManifestSummary {
+    pub id: Exactly32Bytes,
+    pub summary: ManifestSummary,
+}
+
+impl SignableManifestSummary {
+    pub fn new(summary: ManifestSummary) -> Self {
+        Self {
+            id: Exactly32Bytes::generate(),
```

Comment:
prefer using `uuid`, which is what we want here, right? No need for a cryptographically secure bytes?

Reply:
I think we want `impl From<ManifestSummary> for SignableManifestSummary` using this ctor?